src/app/(proper_react)/redesign/(authenticated)/user/dashboard/subscribed/page.tsx

```diff
@@ -36,16 +37,21 @@ export default function Subscribed() {
             `/redesign/user/dashboard/fix/data-broker-profiles/welcome-to-premium`,
           );
         } else {
-          router.replace(`/`);
+          const message = "No subscription found";
+          captureMessage(message);
+          console.error(message);
+          router.replace("/redesign/user/dashboard");
         }
       } catch (ex) {
-        console.error(ex);
         captureException(ex);
-        router.replace(`/`);
+        console.error(ex);
+        router.replace("/redesign/user/dashboard");
       }
     }
     void updateSession();
-  }, [update, router]);
+    // Only run once, since we will be immediately redirecting no matter what.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
@Vinnl and I chatted about this one when it landed - I started with `[]` since I want this to run only once (twice in dev mode but that's OK), the problem is that `update` causes this to run over and over, when we only ever want to run this a single time. It will explicitly redirect somewhere no matter what happens (and if it somehow fails we don't want it to keep running in a loop either)